cleo/io/outputs/stream_output.py

```diff
@@ -107,7 +107,7 @@ def _has_color_support(self) -> bool:
             # Checking for Windows version
             # If we have a compatible version
             # activate color support
-            windows_version = sys.getwindowsversion()
+            windows_version = sys.getwindowsversion()  # type: ignore[attr-defined]
```

Comment:
`mypy` has problems with more complex platform-specific checks. The same is in line 119.

Reply:
i think it should be safe to just change the condition to `sys.platform == "win32"`, which mypy will understand